tests/test_planning.py

```diff
@@ -601,3 +602,52 @@ def _slow_policy(atoms, objs, goal):
             max_horizon=CFG.horizon,
             abstract_policy=_slow_policy,
             max_policy_guided_rollout=50)
+
+
+@pytest.mark.parametrize(
```

Comment:
+1 for doing a little bit of testing. I don't care enough to make you change it, but I would argue that this is a case where the parametrize pytest thingamabob is actually making this test clunkier and harder to understand